.circleci/config.yml

```diff
@@ -1244,12 +1269,31 @@ jobs:
                   test/externalTests/<<parameters.project>>.sh solcjs /tmp/workspace/soljson.js
       - when:
           condition:
-            equal: [<< parameters.binary_type >>, "native"]
+            and:
+                - equal: [<< parameters.binary_type >>, "native"]
+                - equal: [<< parameters.script_lang >>, "shellscript"]
           steps:
             - run:
                 name: External <<parameters.project>> tests (native)
                 command: |
                   test/externalTests/<<parameters.project>>.sh native /tmp/workspace/solc/solc
+      - when:
+          condition:
+            and:
+                - equal: [<< parameters.binary_type >>, "native"]
+                - equal: [<< parameters.script_lang >>, "python3"]
+          steps:
+            - run:
+                name: Install script dependencies
+                command: |
+                    sudo apt update
+                    sudo apt install python3-pip --assume-yes --no-install-recommends
+                    python3 -m pip install requests --user
+            - run:
+                name: External <<parameters.project>> tests (native)
+                command: |
+                    test/externalTests/<<parameters.project>>.py --solc-binary-type native --solc-binary-path /tmp/workspace/solc/solc
+                no_output_timeout: 30m
```

Comment:
Oh, so I guess this means foundry does no print anything when it's running tests?

BTW, is it really that slow? I expected Foundry tests to be way faster than the Hardhat ones.

Reply:
Yeah, all the more reason not to do foundry installation in the script then :)

By the way, it would be fine to install it from official binaries instead of building it. With caching it will be less of a waste but still, there's no need to build it on our side if we don't have to. It would actually be best to get it from distro repos, but unfortunately last I checked it did not seem be be packaged.